Index: opencv46/build/intermediates/cxx/create_cxx_tasks_21_timing.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 20ms\ncreate_cxx_tasks completed in 27ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 38ms\ncreate_cxx_tasks completed in 48ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 25ms\ncreate_cxx_tasks completed in 31ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model\n    create-ARMEABI_V7A-model 11ms\n    [gap of 27ms]\n  create-initial-cxx-model completed in 43ms\n  [gap of 11ms]\ncreate_cxx_tasks completed in 54ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 20ms\ncreate_cxx_tasks completed in 26ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 46ms\n  [gap of 13ms]\ncreate_cxx_tasks completed in 59ms\n\n# C/C++ build system timings\ncreate_cxx_tasks\n  create-initial-cxx-model 40ms\ncreate_cxx_tasks completed in 41ms\n\n
===================================================================
diff --git a/opencv46/build/intermediates/cxx/create_cxx_tasks_21_timing.txt b/opencv46/build/intermediates/cxx/create_cxx_tasks_21_timing.txt
--- a/opencv46/build/intermediates/cxx/create_cxx_tasks_21_timing.txt	
+++ b/opencv46/build/intermediates/cxx/create_cxx_tasks_21_timing.txt	
@@ -38,3 +38,13 @@
   create-initial-cxx-model 40ms
 create_cxx_tasks completed in 41ms
 
+# C/C++ build system timings
+create_cxx_tasks
+  create-initial-cxx-model 42ms
+create_cxx_tasks completed in 45ms
+
+# C/C++ build system timings
+create_cxx_tasks
+  create-initial-cxx-model 30ms
+create_cxx_tasks completed in 31ms
+
